app-admin/ananicy/ananicy-2.2.1-r1.ebuild

```diff
@@ -0,0 +1,36 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+PYTHON_COMPAT=( python3_{7,8,9} pypy3 )
+
+inherit python-single-r1
+
+DESCRIPTION="ANother Auto NICe daemon"
+HOMEPAGE="https://github.com/Nefelim4ag/Ananicy"
+SRC_URI="https://github.com/Nefelim4ag/Ananicy/archive/${PV}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${P^}"
+LICENSE="GPL-3"
```

Comment:
```suggestion

LICENSE="GPL-3"
```

as per skel.ebuild

Reply:
I'm using `S` that stay in between `SRC_URI` and `LICENSE`

Reply:
I'm saying put a new line before LICENSE?